python/ray/autoscaler/_private/monitor.py

```diff
@@ -161,6 +159,12 @@ def __init__(self,
         worker.redis_client = self.redis
         gcs_client = GcsClient.create_from_redis(self.redis)
         _initialize_internal_kv(gcs_client)
+        if monitor_ip:
+            monitor_addr = f"{monitor_ip}:{AUTOSCALER_METRIC_PORT}"
+            self.redis.set("AutoscalerMetricsAddress", monitor_addr)
```

Comment:
Why not put this in an else branch where `use_gcs_for_bootstrap()` is `False`, or add a comment?

Reply:
That's because it's used in monitors right now. I'll remove this in this PR https://github.com/pulls